### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by HasaluSL**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by ParthS007** [comment]
@HasaluSL Please provide a preview link !

**Comment by HasaluSL** [comment]
https://introductorymusicblockstutorial.wordpress.com/2017/12/10/interview-of-a-teacher/

**Comment by lavavi** [comment]
https://bloglavavi.wordpress.com/

File: _data/blogs.yml [hunk 1]
```diff
@@ -82,3 +82,8 @@
   title: My Interaction with FOSSASIA
   img: tushar.jpg
   author_name: Tushar Kumar
+   
+- url: https://introductorymusicblockstutorial.wordpress.com/2017/12/10/interview-of-a-teacher/
+  title: Interview with a teacher
+  img: 2.jpg
```
Comment (anshumanv) [inline _data/blogs.yml]: Please upload your own image for your post.

**Comment by ParthS007** [comment]
Please resolve conflicts and do the suggested changes .

**Comment by akshatnitd** [comment]
@HasaluSL Please resolve the conflicts :)

**Comment by HasaluSL** [comment]
okay

**Comment by harshit98** [comment]
@HasaluSL Kindly add your details in `blogs.yml`.

**Comment by anshumanv** [comment]
Duplicate of #278

**Pull request closed without merging**